dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/RoleChecks.ts

```diff
@@ -51,6 +50,7 @@ describeIf(
       cy.ensureLoggedIn(reader_name, reader_pw);
       readerAndUploaderPages.forEach((page) => {
         it(`Non uploader should be able to access ${page}`, () => {
+          cy.visit(page);
```

Comment:
Why do we need this now?

Reply:
It was simply forgotten in main.  Meaning that this test never actually visited any page in this part 😆 .

Reply:
Thats hilariou xD